### DISCUSSION TITLE
直播勋章快速更换是否已经失效？

### DISCUSSION THREAD

**Discussion by Kal-tsit63** [the1812/Bilibili-Evolved#3515] [提问 / Q&A] [answered]
Microsoft Edge
版本 103.0.1264.62 (正式版本) (64 位)

Tampermonkey
v4.18.6165

Bilibili Evolved
v2.2.2-7-g095ddcc03

侧边栏点击更换勋章与更换头衔完全无反应。                         
想请问一下，是否已经失效？
Commit: 95fbb00c7
内部名称: badgeHelper

**Comment by the1812** [comment, marked as answer]
~~是的~~

v2.2.4 已修复
